proto/exocore/operator/v1/query.proto

```diff
@@ -16,6 +16,42 @@ message GetOperatorInfoReq {
   [(cosmos_proto.scalar) = "cosmos.AddressString"];
 }
 
+// QueryOperatorUSDValueRequest is the request to obtain the USD value for operator.
+message QueryOperatorUSDValueRequest {
+  // operator_addr is the operator address,its type should be a sdk.AccAddress
```

Comment:
can this two fieleds can be extracted into public fields as the following ?

message AddressInfo {
  string operator_addr = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
  string avs_address = 2 [(gogoproto.customname) = "AVSAddress"];
}

message QueryOperatorUSDValueRequest {
  AddressInfo address_info = 1;
}

message QueryAVSUSDValueRequest {
  string avs_address = 1 [(gogoproto.customname) = "AVSAddress"];
}

message QueryOperatorSlashInfoRequest {
  AddressInfo address_info = 1;
}

Reply:
Yeah, it's better.

Reply:
Yeah, I will rename it.